lib/RESTClient.js

```diff
@@ -4,10 +4,18 @@ const assert = require('assert');
 const http = require('http');
 const https = require('https');
 const querystring = require('querystring');
-const werelogs = require('werelogs');
 
+const HttpAgent = require('agentkeepalive');
+const HttpsAgent = require('agentkeepalive').HttpsAgent;
```

Comment:
Can this be?
```suggestion
const HttpsAgent = HttpAgent.HttpsAgent;
```

Reply:
I don't know, since I don't have a https setup to check and rely on Integration for https validation, and since the module documentation uses this form, I'd rather leave it as is in doubt. I haven't seen this way of nesting before but it's possible that there is special handling by the module when doing a simple `HttpAgent = require(..)` to return the appropriate class.

Reply:
Ok fair, some of the require / import things in JS are still a little fuzzy to me.